plasma_framework/test/src/transactions/GenericTransaction.test.js

```diff
@@ -4,17 +4,17 @@ const { BN, constants, expectRevert } = require('openzeppelin-test-helpers');
 
 const { PaymentTransaction, PaymentTransactionOutput } = require('../../helpers/transaction.js');
 
-const WireTransaction = artifacts.require('WireTransactionWrapper.sol');
+const GenericTransaction = artifacts.require('GenericTransactionWrapper.sol');
 
-contract('WireTransaction', () => {
+contract('GenericTransaction', () => {
```

Comment:
Some comments on this test file:
1. Remove the test of `getTransactionType`. It is no longer a function of the library.
1. Add tests for `decode` and `decodeOutput`. I think some of them are currently covered in the `PaymentTransactionModel` tests. Please consider move those tests related to `GenericTransaction` here or duplicate the tests.